<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>606</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>621</width>
    <height>606</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>621</width>
    <height>606</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>101</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QLCDNumber" name="lcdMin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdSec">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>471</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblWait">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QToolButton" name="btnPause">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>496</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblPaused">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>190</y>
      <width>401</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>PAUSED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionTrivial"/>
     <addaction name="actionEasy"/>
     <addaction name="actionMedium"/>
     <addaction name="actionHard"/>
     <addaction name="actionExpert"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionResume"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCheat">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionRetry"/>
    <addaction name="actionSolve"/>
    <addaction name="actionShowMistakes"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuCheat"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSolve">
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="actionShowMistakes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Mistakes</string>
   </property>
  </action>
  <action name="actionTrivial">
   <property name="text">
    <string>Trivial</string>
   </property>
  </action>
  <action name="actionEasy">
   <property name="text">
    <string>Easy</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="text">
    <string>Hard</string>
   </property>
  </action>
  <action name="actionExpert">
   <property name="text">
    <string>Extreme</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo(Ctrl+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo(Ctrl+Shift+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load Game...</string>
   </property>
  </action>
  <action name="actionRetry">
   <property name="text">
    <string>Retry</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Game</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="text">
    <string>Resume Game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
